From 9ba7f1d1857f0a5a28feea7b89f94c86d5a135c8 Mon Sep 17 00:00:00 2001
From: hmtheboy154 <buingoc67@gmail.com>
Date: Sun, 13 Oct 2024 05:33:24 +0700
Subject: [PATCH] Test switching to blivet-gui

Signed-off-by: hmtheboy154 <buingoc67@gmail.com>
---
 Dockerfile | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/Dockerfile b/Dockerfile
index ea77230..c77acc3 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -25,8 +25,11 @@ RUN apt install grub-common grub2-common grub-pc-bin grub-efi-ia32-bin grub-efi-
 # As an extra, install gxmessage and consolekit for shutdown
 RUN apt install gxmessage consolekit -y
 
-# Gparted
-RUN apt install gparted -y
+# blivet-gui
+RUN echo 'deb http://download.opensuse.org/repositories/home:/vtrefny/Debian_Testing/ /' | tee /etc/apt/sources.list.d/home:vtrefny.list
+RUN curl -fsSL https://download.opensuse.org/repositories/home:vtrefny/Debian_Testing/Release.key | gpg --dearmor | tee /etc/apt/trusted.gpg.d/home_vtrefny.gpg > /dev/null
+RUN apt update
+RUN apt install blivet-gui -y
 
 # Calamares
 RUN apt install calamares calamares-extensions calamares-extensions-data -y
-- 
2.47.0

